packages/paste-core/components/inline-code/stories/index.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import {InlineCode} from '../src';
+import type {Story} from '@storybook/react';
+import {Text} from '@twilio-paste/text';
+import {Box} from '@twilio-paste/box';
+import {Stack} from '@twilio-paste/stack';
+import {useTheme} from '@twilio-paste/theme';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Inline Code',
+  component: InlineCode,
+};
+
+export const Default: Story = () => {
+  return <InlineCode>0000001111111122234444444</InlineCode>;
+};
+
+export const ReallyLongContent: Story = () => {
+  return (
+    <Box maxWidth="size40">
+      <InlineCode>
+        000 00 0111111 1122 2344444 44 000 00 0111111 1122 2344444 44 000 00 0111111 1122 2344444 44 000 00 0111111 1122
+        2344444 44
+      </InlineCode>
+    </Box>
+  );
+};
+
+export const InlineWithText: Story = () => {
+  return (
+    <Text as="span">
+      If the composite has a single row or column (one-dimensional), the <InlineCode>orientation</InlineCode> value
+      determines which arrow keys can be used to move focus.
```

Comment:
suggestion(non-blocking): Use a Twilio example here, for example, from Discussion #2461 :

> Your `{{friendly_name}}` verification code is: `{{code}}`.

Reply:
The specific syntax ( the '{{' ) that this example uses makes it really really difficult to use in the live preview for our site. Let me know if you've got another example in mind @serifluous ; I'll keep this one for now.

Reply:
added!